simplified-app-shared/src/main/java/org/nypl/simplified/app/services/SimplifiedServices.kt

```diff
@@ -32,12 +34,7 @@ import org.nypl.simplified.analytics.api.Analytics
 import org.nypl.simplified.analytics.api.AnalyticsConfiguration
 import org.nypl.simplified.analytics.api.AnalyticsEvent
 import org.nypl.simplified.analytics.api.AnalyticsType
-import org.nypl.simplified.app.AdobeDRMServices
-import org.nypl.simplified.app.Bugsnag
-import org.nypl.simplified.app.BundledContentResolver
-import org.nypl.simplified.app.NetworkConnectivityType
-import org.nypl.simplified.app.R
-import org.nypl.simplified.app.ScreenSizeInformationType
+import org.nypl.simplified.app.*
```

Comment:
:nauseated_face:

Reply:
How does this keep getting added! I need to set my IDE to stop doing this.

Reply:
I use this setting:

![settings](https://user-images.githubusercontent.com/612494/64359617-42bc8180-cff8-11e9-956b-d3a3e1095b78.png)

However, even that doesn't fully prevent it. The IDE will still insert start imports, it's just that the code formatter will clean them up afterwards if you ask it to. It seems to be specific to Kotlin in the IDE, doesn't happen in Java.

Reply:
Will that linting tool in your other PR catch these? I'd be surprised if there wasn't a feature to disallow them.

Reply:
Ktlint will catch these, and I will use this setting too